setup.py

```diff
@@ -65,9 +65,9 @@ def run(self) -> None:
     ],
     python_requires=">=3.8",
     install_requires=[
-        "sqlparse>=0.3.1",
+        "sqlparse==0.4.4",
         "networkx>=2.4",
-        "sqlfluff>=1.4.5",
+        "sqlfluff==2.0.3",
```

Comment:
Let's still use `>=` and not pin them to specific version. These breaking change happens rarely.

Reply:
If that `LATERAL VIEW` thing is breaking with sqlfluff>2.0.3, I'm OK pin it to 2.0.3 for now.

Reply:
Exactly, that is an issue with `sqlfluff>2.0.3`, so we can leave as it is for now.